modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -347,29 +338,34 @@ trait ObservationOptics { self: ObservationModel.type =>
   val activeStatus: Lens[ObservationModel, ObsActiveStatus] =
     Lens[ObservationModel, ObsActiveStatus](_.activeStatus)(a => _.copy(activeStatus = a))
 
-  val pointing: Lens[ObservationModel, Option[Either[Asterism.Id, Target.Id]]] =
-    Lens[ObservationModel, Option[Either[Asterism.Id, Target.Id]]](_.pointing)(a => _.copy(pointing = a))
+  val targets: Lens[ObservationModel, TargetEnvironmentModel] =
+    Lens[ObservationModel, TargetEnvironmentModel](_.targets)(a => _.copy(targets = a))
```

Comment:
Would it make sense to use the `Focus` macro?

Reply:
Thanks, I'll switch.  In general I prefer more explicit code even at the cost of boilerplate but `Focus` (which is just `GenLens`?) seems fine.